src/cases/containers/case-list/case-list.component.ts

```diff
@@ -94,6 +112,11 @@ export class CaseListComponent implements OnInit, OnDestroy {
       this.jurisdictions = jurisdictions;
     });
 
+    this.featureToggleSubscription = this.featureToggleService.getValue('shareable-jurisdictions', []).subscribe(value => {
+      this.shareableJurisdictions = value;
+      this.setupCaseShareVisibility();
```

Comment:
Agreed that the two aysch pipes cause the errors, but the 'this.setupCaseShareVisibility();' and another one in line 156 would probably cause race conditions when we put this trigger(this.setupCaseShareVisibility) in two subscription blocks. Ideally we need to merge or switch the two observables into one isCaseShareVisible$ and still use the asynch pipe like (isCaseShareVisible$ | asynch) in the html.

Reply:
I don't follow why it would cause race conditions. The `setupCaseShareVisibility()` method checks for the presence of data from both subscriptions before proceeding. I'm not entirely sure what would be gained from putting an async pipe in the HTML...?